odd-platform-api/src/main/resources/db/migration/V0_0_87__create_relation_tables.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE IF NOT EXISTS relationship
+(
+    id                         bigserial PRIMARY KEY,
+    name                       varchar(512),
+    data_source_id             bigint,
+    relationship_oddrn         varchar(512) UNIQUE,
+    source_dataset_oddrn       varchar(512)       NOT NULL,
+    target_dataset_oddrn       varchar(512)       NOT NULL,
+    is_manualy_created         boolean,
+    relationship_status        SMALLINT,
+    last_ingested_at           TIMESTAMP WITHOUT TIME ZONE,
+    deleted_at                 TIMESTAMP WITHOUT TIME ZONE,
+
+    CONSTRAINT relationship_fk_data_source FOREIGN KEY (data_source_id) REFERENCES data_source (id)
+);
```

Comment:
it's polymorphic one-to-one relation between `relationships ` table and `erd_relationship`, `graph_relationship` tables. We should add some `column_type` that defines it.

Reply:
done